server/superdesk/errors.py

```diff
@@ -122,7 +122,14 @@ class UserInactiveError(SuperdeskApiError):
     """User is inactive, access restricted"""
     status_code = 403
     payload = {'is_active': False}
-    message = 'Account suspended, access restricted.'
+    message = 'Account is inactive, access restricted.'
+
+
+class UserDisabledError(SuperdeskApiError):
```

Comment:
Errors specific to endpoints should be defined in their corresponding package. Now if you remove ingest for example the errors are still there.